node.gyp

```diff
@@ -184,12 +184,15 @@
       'defines': [
         'NODE_ARCH="<(target_arch)"',
         'NODE_PLATFORM="<(OS)"',
-        'NODE_TAG="<(node_tag)"',
         'NODE_V8_OPTIONS="<(node_v8_options)"',
         'NODE_WANT_INTERNALS=1',
       ],
 
+
       'conditions': [
+        [ 'node_tag!=""', {
```

Comment:
move this here because our binaries aren't being compiled with a `-pre` tag for non-release builds (see src/node_version.h), I don't know when or how this happened but I noticed it long ago for io.js.